scRNA-seq/post-processing_scRNA-seq.Rmd

```diff
@@ -113,52 +70,67 @@ seurat_norm <- NormalizeData(object = seurat_obj,
 ```
 
 #### Identify variable genes and scale the data
-In this step, we will identify genes with higher variance. 
-These are probably the genes that are of more interest to us and they are what 
-we will use for Principal Components Analysis in the upcoming step.
-```{r Find Variable Genes}
-seurat_norm <- FindVariableGenes(object = seurat_norm, mean.function = ExpMean,
-                                x.low.cutoff = 0.0125, x.high.cutoff = 3,
-                                y.cutoff = 0.5, do.plot = FALSE)
 
+As we mentioned, there are mores genes that have zeroes in single-cell RNA-seq 
+data but our filtering step has hopefully removed all of these low-to-no 
+expression genes.
+However, the rest of the transcriptome is usually overwhelmed by a few genes that 
+are highly expressed, but don't change much with different circumstances (think 
+housekeeping-like genes). 
+These housekeeping-like genes also are unlikely to be helpful for us in looking 
+for biologically meaningful gene expression patterns. 
+To avoid our analyses being watered down by these low variance genes, we want to
+identify a subset of high variance genes that may be more informative about the 
+differences between cells. 
+The Seurat function `FindVariableGenes` will do this for us and print out 
+a expression by variance plot so we can get an overall picture of the genes 
```

Comment:
You've set `do.plot = FALSE` in `FindVariableGenes`. I would not reference printing out a plot here.

Reply:
Whoops that's a remnant.